### DISCUSSION TITLE
SpoofDPI на роутере с Entware

### DISCUSSION THREAD

**Discussion by gray1717** [xvzc/SpoofDPI#187] [Q&A]
https://github.com/gray1717/spoofDPI_to_router пробуйте, запилил скрипт для установки и сборки пакета *.ipk  для пакетного менеджера opkg из Entware. У меня asus rt-ac86u на aarch64, пока работает.
Данный способ подойдет если нужно настроить proxy на устройстве, на которое никакой софт не установить, в этом случае роутер будет подходящим устройством для установки spoof-dpi

**Comment by Rvcube** [comment]
А если такое же на OpenWRT? Реально?

**Reply by gray1717** [reply]
реально, только мне проверить неначем
... позже посмотрю как там все устроено, думаю почти ничего не нужно переделывать )
что у вас команда ````uname -a```` выдает?

**Reply by Rvcube** [reply]
> реально, только мне проверить неначем
> ... позже посмотрю как там все устроено, думаю почти ничего не нужно переделывать )
> что у вас команда ````uname -a```` выдает?

aarch64

**Reply by vehametra** [reply]
Это MIPS правда. Но препятсвий не вижу.
![Снимок экрана_2024-10-11_17-35-05](https://github.com/user-attachments/assets/95a28930-0bfa-4980-9840-eb56b567a084)
